src/RamlToOpenApiConverter/Constants.cs

```diff
@@ -3,5 +3,7 @@
     internal static class Constants
     {
         public const string IncludeTag = "!include";
+        public const string Is_tag = "is";
```

Comment:
Just name this `IsTag`

Reply:
https://github.com/StefH/RamlToOpenApiConverter/pull/18/commits/622e385ae6751d91e84de904a27ee2f708dddcf8